OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,18 @@
+//
+//  CQCategory.h
+//  Unit1WeekFourHomework
+//
+//  Created by Jackie Meggesto on 8/12/15.
+//  Copyright (c) 2015 Jackie Meggesto. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
+
+@property (nonatomic) NSString *name;
+@property (nonatomic) NSArray *options;
+@property (nonatomic) NSString *selection;
+@property (nonatomic) NSIndexPath *cqIndexPath;
```

Comment:
I haven't gotten to your controller files yet, but your data model probably shouldn't have to know anything about index paths. I'll add additional comments if I see this come up later on.